src/examples/real.lean

```diff
@@ -14,8 +19,14 @@ open liquid_tensor_experiment category_theory category_theory.limits opposite
 
 example : conditionally_complete_linear_order ℝ := infer_instance
 example : linear_ordered_field ℝ := infer_instance
+example : conditionally_complete_linear_ordered_field ℝ := infer_instance
 
 example : ℝ≥0 = {r : ℝ // r ≥ 0} := rfl
 
--- TODO: once we bump mathlib past [#3292](https://github.com/leanprover-community/mathlib/pull/3292) we can show that `ℝ` is the unique
--- conditionally complete linearly ordered field, up to isomorphism.
+-- Any conditionally complete linear ordered field is isomorphic (as an ordered ring) to `ℝ`.
+example {R : Type*} [conditionally_complete_linear_ordered_field R] : R ≃+*o ℝ := default
+
+-- The isomorphism above is unique
+example {R : Type*} [conditionally_complete_linear_ordered_field R] (e₁ e₂ : R ≃+*o ℝ) :
+  e₁ = e₂ :=
+(eq_iff_true_of_subsingleton _ _).mpr trivial
```

Comment:
Is this not just `subsingleton.elim e₁ e₂`?

Reply:
This was the handy work of library search. I changed it ;)